### DISCUSSION TITLE
Align Text widget with leading button on center in Flutter

### DISCUSSION THREAD

**Discussion by topdevOscar** [community/community#52743] [Programming Help]
### Body

Hello,
I ran into some issues while developing mobile app using Flutter/Dart.
There is a back button, and title in the same line at the top of screen. (Not AppBar)
The back button(leading title) is on left, and the title should be on center.
Problem is, I can't put the title on center exactly.
Here's my code.

I have been looking for a solution for hours but didn't find yet.
Please help me.

```
Column(
  mainAxisAlignment: MainAxisAlignment.start,
  children: [
    Row(
      children: [
          IconButton(
            padding: const EdgeInsets.all(0),
            icon: const Icon(
              Icons.arrow_back,
            ),
            onPressed: onBackPressed != null? () {
              onBackPressed!();
            } : null,
          ),
        Expanded(
          child: Text(title, textAlign: TextAlign.center),
        ),
        if (tailAction != null)
          tailAction!
      ],
    ),
)
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Salmerrr** [comment]
> 
<details><summary>Details</summary>
<p>



</p>
</details>
